nixos/modules/hardware/bitcraze.nix

```
@@ -0,0 +1,23 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.hardware.bitcraze;
+
+in
+{
+  options.hardware.bitcraze = {
+    enable = lib.mkOption {
+      type = lib.types.bool;
+      default = false;
+      description = lib.mdDoc ''
+        Enables Bitcraze Crazyflie udev rules and ensures 'plugdev' group exists.
+        This is a prerequisite to using Bitcraze Crazyflie devices without being root, since Bitcraze Crazyflie USB descriptors will be owned by plugdev through udev.
+      '';
+    };
```

`mkEnableOption` prepends the content with "Whether to enable", which is why the first sentence might look a bit weird.

```suggestion
    enable = lib.mkEnableOption (lib.mdDoc ''
      Bitcraze Crazyflie udev rules and ensure 'plugdev' group exists.
      
      This is a prerequisite to using Bitcraze Crazyflie devices without being root, since Bitcraze Crazyflie USB descriptors will be owned by plugdev through udev.
    '');

```